examples/pa_engine_single_unit_linked_templated_component_example.py

```diff
@@ -0,0 +1,254 @@
+            parent_template_id=parent_template_id,
+            description="This is a templated PA component",
+            component_data = PAComponentData(
+                accounts = [PAIdentifier(id=portfolio)],
+                benchmarks = [PAIdentifier(id=benchmark)],
+                columns = columns,
+                groups = groups,
+                currencyisocode = currencyisocode,
+                componentdetail = componentdetail
+            )
+        )
+
+        templated_pa_component_parameters_root = TemplatedPAComponentParametersRoot(
+            data = templated_pa_component_parameters
+        )
+
+        templated_pa_components_api = TemplatedPAComponentsApi(api_client)
+
+        response = templated_pa_components_api.create_templated_pa_components(
+            templated_pa_component_parameters_root = templated_pa_component_parameters_root)
+
+        component_id = response[0].data.get("id")
+
+        print("PA Component Id: " + component_id)
+
+        # do PA calculation with given templated component id
+        pa_accounts = [PAIdentifier(id=portfolio)]
+        pa_benchmarks = [PAIdentifier(id=benchmark)]
+        pa_dates = PADateParameters(
+            startdate=startdate, enddate=enddate, frequency=frequency)
+
+        pa_calculation_parameters = {"1": PACalculationParameters(componentid=component_id, accounts=pa_accounts,
+                                                                  benchmarks=pa_benchmarks, dates=pa_dates)}
+
+        pa_calculation_parameter_root = PACalculationParametersRoot(
+            data=pa_calculation_parameters)
+
+        pa_calculations_api = PACalculationsApi(api_client)
+
+        post_and_calculate_response = pa_calculations_api.post_and_calculate(
+            pa_calculation_parameters_root=pa_calculation_parameter_root)
+
+        # comment the above line and uncomment the below line to run the request with the cache_control header defined earlier
+        # post_and_calculate_response = pa_calculations_api.post_and_calculate(pa_calculation_parameters_root=pa_calculation_parameter_root, cache_control=cache_control)
+        if post_and_calculate_response[1] == 201:
+            output_calculation_result(post_and_calculate_response[0]['data'])
+        elif post_and_calculate_response[1] == 200:
+            for (calculation_unit_id, calculation_unit) in post_and_calculate_response[0].data.units.items():
+                print("Calculation Unit Id:" +
+                      calculation_unit_id + " Failed!!!")
+                print("Error message : " + str(calculation_unit.errors))
+        else:
+            calculation_id = post_and_calculate_response[0].data.calculationid
+            print("Calculation Id: " + calculation_id)
+
+            status_response = pa_calculations_api.get_calculation_status_by_id(id=calculation_id)
+
+            while status_response[1] == 202 and (status_response[0].data.status in ("Queued", "Executing")):
```

Comment:
Any specific reason for adding a check on status too? I think that status_code check is good enough.

Reply:
I was following the formatting of `pa_engine_single_unit_example.py` and `pa_engine_multiple_unit_example.py` ([example](https://github.com/factset/analyticsapi-engines-python-sdk/blob/master/examples/pa_engine_single_unit_example.py#L89)), but I agree that `status_code` is enough. Removed the extra status check here https://github.com/factset/analyticsapi-engines-python-sdk/pull/62/commits/8dddeccde21189ee9282493fd177953993d8793e